sw/device/lib/dif/dif_adc_ctrl.h

```diff
@@ -98,14 +99,15 @@ typedef enum dif_adc_ctrl_irq_cause {
   /**
    * Sample ready cause in Oneshot mode.
    */
-  kDifAdcCtrlIrqCauseOneshot = 1U << 8,
+  kDifAdcCtrlIrqCauseOneshot = 1U << ADC_CTRL_ADC_INTR_STATUS_ONESHOT_BIT,
   /**
    * All IRQ causes ORed together.
    *
    * This is useful when clearing all IRQ causes at once, to initialize the ADC
    * Controller.
    */
-  kDifAdcCtrlIrqCauseAll = (1U << 9) - 1,
+  kDifAdcCtrlIrqCauseAll =
```

Comment:
nit: would it hurt if we used `UINT32_MAX` here?

Reply:
If we did that then we can't use utilities like `bitfield_popcount32` to return the number of IRQ causes.